### ISSUE TITLE
Inline Edit Bug: frontend currency type

### ISSUE THREAD

**Issue by sbani** [orocrm/platform#468]
Hi,

I found a bug while using the inline edit functionality for currency.
We use the german language where you input `200.003,20` instead of `200,003.20`.

But when you try to input `,` during inline edit you see the error:
> Please enter a valid number.

Is there a way to fix it?

**Comment by artml** [comment]
Hi Sufijen,

A bug can always be fixed :) would you like to contribute the fix yourself or wait for our fix?

This is most likely a problem with frontend validation of the control.

**Comment by sbani** [comment]
thank you @artml for your fast response! I will update here on monday if I find something or not. I don't have that much time for research for this topic.

**Comment by sbani** [comment]
Update: I have similar problems with the `datetime` frontend type. I will spent a short time for research and will post my findings

**Comment by sbani** [comment]
I think you are using the [wrong](https://github.com/orocrm/platform/blob/master/src/Oro/Bundle/FormBundle/Resources/public/js/app/views/editor/number-editor-view.js#L56) number formatter.
The correct one:
```js
require('orodatagrid/js/datagrid/formatter/number-formatter')
```
But I really don't get why there are 3 number formatters in the project which do basically the same.
And I don't find a solution for the js validator because it's always returning that the input is invalid

**Comment by artml** [comment]
I see. Thank you for the report, we will be fixing it soon. (Hard to tell _how_ soon, but I will update the issue accordingly.)

**Comment by artml** [comment]
Fixed in 1.10 release
